rotkehlchen/tests/unit/decoders/test_shutter.py

```diff
@@ -0,0 +1,141 @@
+    )
+    timestamp = TimestampMS(1706703155000)
+    gas_amount_str = '0.006946508605618104'
+    expected_events = [
+        EvmEvent(
+            tx_hash=tx_hash,
+            sequence_index=0,
+            timestamp=timestamp,
+            location=Location.ETHEREUM,
+            event_type=HistoryEventType.SPEND,
+            event_subtype=HistoryEventSubType.FEE,
+            asset=A_ETH,
+            balance=Balance(amount=FVal(gas_amount_str)),
+            location_label=ethereum_accounts[0],
+            notes=f'Burned {gas_amount_str} ETH for gas',
+            counterparty=CPT_GAS,
+        ), EvmEvent(
+            tx_hash=tx_hash,
+            sequence_index=132,
+            timestamp=timestamp,
+            location=Location.ETHEREUM,
+            event_type=HistoryEventType.INFORMATIONAL,
+            event_subtype=HistoryEventSubType.NONE,
+            asset=A_SHU,
+            balance=Balance(amount=FVal('1086.95652173913')),
+            location_label=ethereum_accounts[0],
+            notes='Claim 1086.95652173913 SHU from shutter airdrop and deposit it to 0x22714964e6b9F17798A9e1AD3f2BAb87279876FC',  # noqa: E501
+            counterparty=CPT_SHUTTER,
+            address=SHUTTER_AIDROP_CONTRACT,
+        ),
+    ]
+    assert events == expected_events
+
+
+@pytest.mark.vcr()
+@pytest.mark.parametrize('ethereum_accounts', [['0x36bE40f9bd613dFf47294E7e10D4cae072E06A2D']])
+def test_shu_delegation_from_self(database, ethereum_inquirer, ethereum_accounts):
+    tx_hex = deserialize_evm_tx_hash('0x46a476b09c85d2278f1ac889e943d7f47d029d0985719cc2a73d589a73cb2473')  # noqa: E501
+    tx_hash = deserialize_evm_tx_hash(tx_hex)
+    events, _ = get_decoded_events_of_transaction(
+        evm_inquirer=ethereum_inquirer,
+        database=database,
+        tx_hash=tx_hex,
+    )
+    timestamp = TimestampMS(1705596215000)
+    gas_amount_str = '0.00549203889835413'
+    expected_events = [
+        EvmEvent(
+            tx_hash=tx_hash,
+            sequence_index=0,
+            timestamp=timestamp,
+            location=Location.ETHEREUM,
+            event_type=HistoryEventType.SPEND,
+            event_subtype=HistoryEventSubType.FEE,
+            asset=A_ETH,
+            balance=Balance(amount=FVal(gas_amount_str)),
+            location_label=ethereum_accounts[0],
+            notes=f'Burned {gas_amount_str} ETH for gas',
+            counterparty=CPT_GAS,
+        ), EvmEvent(
+            tx_hash=tx_hex,
+            sequence_index=479,
+            timestamp=timestamp,
+            location=Location.ETHEREUM,
+            event_type=HistoryEventType.INFORMATIONAL,
+            event_subtype=HistoryEventSubType.GOVERNANCE,
+            asset=A_SHU,
+            balance=Balance(),
+            location_label=ethereum_accounts[0],
+            notes=f'Change SHU Delegate of 0x561075538e7B20613Aa0C3fAa8459ac293011584 from {ethereum_accounts[0]} to 0x9Cc9C7F874eD77df06dCd41D95a2C858cd2a2506',  # noqa: E501
```

Comment:
I believe you can remove this `of 0x.. ` part... this one is the vesting contract if I'm not mistaken